www-client/uget/uget-2.2.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit gnome2-utils xdg-utils
+
+IUSE="+gnutls aria2 gstreamer appindicator libnotify nls openssl sockets rss"
+KEYWORDS="~amd64 ~arm ~x86"
+SRC_URI="mirror://sourceforge/urlget/${P}.tar.gz"
+
+DESCRIPTION="Download manager using gtk+ and libcurl"
```

Comment:
Please order variables as in skeleton files as it is otherwise hard to read different ebuilds, i.e.

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE
REQUIRED_USE

RDEPEND
DEPEND

EVERYTHING_ELSE
```